blockchain/blockdao/blockdao_test.go

```diff
@@ -259,6 +259,10 @@ func TestBlockDAO(t *testing.T) {
 			h1, err := dao.GetTipHash()
 			require.NoError(err)
 			require.Equal(h, h1)
+			_, err = dao.GetBlockHash(uint64(4 - i))
```

Comment:
let's use tipHeightBeforeDeletion and tipHashBeforeDeletion instead of "i"

Reply:
fixed in latest commit